docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -14,6 +14,8 @@ Bug fixes
 ~~~~~~~~~
 * Fix handling of keyword arguments in `forecasts.get_processed_data`.
   (:issue:`745`)
+* Fix rounding issue in `clearsky._linearly_scale`.
```

Comment:
Can you add a bit more detail here? Most users won't be aware of this private function.